src/stdfblib/ita/OPCUA_MGR.cpp

```diff
@@ -1323,3 +1324,24 @@ void OPCUA_MGR::parseDestinationName(const std::string& paDestination, std::vect
   }
   paFbName.push_back(dst.substr(0, index));
 }
+
+void OPCUA_MGR::addExtraMgmMethod(const MethodInformation& paMethod){
+  mExtraMgmMethods.emplace_back(paMethod);
```

Comment:
what is the advantage of having that list. These methods only have to be crated once. This could be easily don by overriding the àddExtraMethods` function.

Reply:
the main idea of the lists is that you can have your own `MGR` class easily compose the `OPCUA_MGR` by just adding new  methods. I have another device on my pipeline of devices which needs the `debug` methods plus some more, so they just keep adding methods https://github.com/cochicde/4diac-forte/blob/replay-algorithm/src/stdfblib/ita/replay/ReplayMGR.cpp#L25

It's true that they're used only once, so it should be fine to just remove them all after they're used. Overriding the method sounds like there will be inheritance involved, which would difficult if you want to mix methods from one or other devices.

Reply:
Ah I missed that you don't want to use the OPC UA MGR in an inheritence relation. Thx for the clarification.